src/hidimstat/leave_one_covariate_out.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import pandas as pd
+from joblib import Parallel, delayed
+from sklearn.base import check_is_fitted, clone
+from sklearn.metrics import root_mean_squared_error
+
+from hidimstat.base_perturbation import BasePerturbation
+
+
+class LOCO(BasePerturbation):
+    def __init__(
+        self,
+        estimator,
+        loss: callable = root_mean_squared_error,
+        method: str = "predict",
+        n_jobs: int = 1,
+    ):
+        """
+        Leave-One-Covariate-Out (LOCO) as presented in
+        :footcite:t:`Williamson_General_2023`. The model is re-fitted for each variable/
+        group of variables. The importance is then computed as the difference between
+        the loss of the full model and the loss of the model without the variable/group.
+
+        Parameters
+        ----------
+        estimator : object
+            The estimator to use for the prediction.
+        loss : callable, default=root_mean_squared_error
+            The loss function to use when comparing the perturbed model to the full
+            model.
+        method : str, default="predict"
+            The method to use for the prediction. This determines the predictions passed
+            to the loss function.
+        n_jobs : int, default=1
+            The number of jobs to run in parallel. Parallelization is done over the
+            variables or groups of variables.
+        """
+        super().__init__(
+            estimator=estimator,
+            loss=loss,
+            method=method,
+            n_jobs=n_jobs,
+            n_permutations=1,
+        )
+        self._list_estimators = []
+
+    def fit(self, X, y, groups=None):
+        super().fit(X, y, groups)
```

Comment:
Can you take this opportunity to add a one-line docstring to each private method ?